etcdctl/ctlv3/command/ep_command.go

```diff
@@ -89,8 +89,8 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 			ep := cfg.Endpoints[0]
 			cli, err := v3.New(*cfg)
 			if err != nil {
-				fmt.Printf("%s is unhealthy: failed to connect: %v\n", ep, err)
```

Comment:
oh, I see the patch got rid of the prints; yes, it should be more careful about printing errors